src/main/java/net/sf/rails/game/specific/_1837/StockRound_1837.java

```diff
@@ -93,6 +93,7 @@ protected void setGameSpecificActions() {
                 for (PublicCompany majorComp : comps) {
                     if ((comp.isRelatedToNational(majorComp.getId()))
                         && (majorComp.hasStarted())
+                            // FIXME: getType() will never equal "Major"
```

Comment:
the context information is wrong as we are looking for all PublicCompanies which may be a major or a minor... The check is valid.

Reply:
`majorComp.getType().equals("Major")`

The problem with this conditional check is that `getType()` returns a `CompanyType` object-- hence `CompanyType.equals("Major")` is the method that is actually being called. Since a `String` is being passed into the `equals()` it will never be true